README.rst

```diff
@@ -161,6 +161,43 @@ fault by adding the ``-v`` (verbose) flag::
 
 I need to rebuild this wheel to link with dual-architecture libraries.
 
+Making dual-architecture binaries
+=================================
+
+Modern Mac wheels can be either ``arm64e`` (M1 ARM), ``x86_64`` (64-bit Intel) or
```

Comment:
spurious "e" in `arm64` here?

Reply:
hmm you repeat it below, so I guess it's on purpose - but I've never seen the `e` before

Reply:
I believe `arm64e` refers to some extended features on ARM : see https://stackoverflow.com/questions/52624308/xcode-arm64-vs-arm64e

I found it doing `lipo -archs /usr/bin/python3` on my M2 machine.  I'm guessing that `arm64e` is valid for M1 machines also.  But yes - probably best to stick to `arm64` and maybe do a note about `arm64e`.

Reply:
Ah okay, learned something new. But yes, it's unusual (first time I've seen it) so probably better to stick to plain `arm64`.